packages/vue/src/generators/library/__snapshots__/library.spec.ts.snap

```diff
@@ -110,8 +110,8 @@ exports[`lib should add vue, vite and vitest to package.json 1`] = `
     "@nx/vue": "0.0.1",
     "@swc-node/register": "~1.6.7",
     "@swc/core": "~1.3.85",
-    "@typescript-eslint/eslint-plugin": "^6.9.1",
-    "@typescript-eslint/parser": "^6.9.1",
+    "@typescript-eslint/eslint-plugin": "^6.13.2",
+    "@typescript-eslint/parser": "^6.13.2",
```

Comment:
We should update this test to not use a snapshot - its no fun to have to update it every time we change an unrelated package version. Could be a good follow up PR?

Reply:
I agree. I mentioned it before to @FrozenPandaz, but it hasn't been a priority. There are multiple tests with too broad assertions.